evadb/third_party/databases/postgres/postgres_handler.py

```diff
@@ -108,3 +110,25 @@ def execute_native_query(self, query_string: str) -> DBHandlerResponse:
             return DBHandlerResponse(data=self._fetch_results_as_df(cursor))
         except psycopg2.Error as e:
             return DBHandlerResponse(data=None, error=str(e))
+
+    def _pg_to_python_types(self, pg_type: str):
+        mapping = {
+            "integer": int,
+            "bigint": int,
+            "smallint": int,
+            "numeric": float,
+            "real": float,
+            "double precision": float,
+            "character": str,
+            "character varying": str,
+            "text": str,
+            "boolean": bool,
+            # Add more mappings as needed
+        }
```

Comment:
Why we do pg->python->evadb instead of pg->evadb?

Reply:
Keep evadb types private from third-party integrations.